packages/trader/src/App/Containers/Layout/header.jsx

```diff
@@ -104,7 +94,7 @@ Header.propTypes = {
 
 // need to wrap withRouter around connect
 // to prevent updates on <MenuLinks /> from being blocked
-export default withRouter(connect(
+export default connect(
```

Comment:
could you remove the comments above as well? it's not relevant anymore

Reply:
Sure 👍